.gitignore

```diff
@@ -9,6 +9,7 @@
 /CMakeUserPresets.json
 /build*/
 /install*/
+/out/
```

Comment:
What goes in `out`. Why not put whatever it is under `build`. `out` is a very generic name. Can we use something more specific, e.g. `python-out`?

Reply:
I removed this from .gitignore - not necessary anymore.